clients/src/main/java/org/apache/kafka/clients/admin/internals/ListConsumerGroupOffsetsHandler.java

```diff
@@ -40,16 +40,27 @@ public class ListConsumerGroupOffsetsHandler extends AdminApiHandler.Batched<Coo
 
     private final CoordinatorKey groupId;
     private final List<TopicPartition> partitions;
+    private final boolean requireStable;
     private final Logger log;
     private final AdminApiLookupStrategy<CoordinatorKey> lookupStrategy;
 
+    public ListConsumerGroupOffsetsHandler(
+            String groupId,
```

Comment:
nit: Indentation seems to be off in comparison to the other constructor/methods.